.github/workflows/ci.yml

```diff
@@ -24,7 +24,5 @@ jobs:
       run: npm run build
     - name: i18n_extract
       run: npm run i18n_extract
-    - name: is-es5
-      run: npm run is-es5
```

Comment:
This is no longer included in `frontend-template-application` ([here](https://github.com/openedx/frontend-template-application/blob/master/package.json)) and I don't think we need it, but LMK if that seems wrong.

Reply:
Seems okay to me.